web/src/modules/common/components/Page.js

```diff
@@ -53,7 +54,10 @@ const Page = ({
       {defaultThumbnailSrc && <Thumbnail alt='' src={defaultThumbnailSrc} srcSet={thumbnailSrcSet} />}
       <Caption title={title} />
       {children}
-      <RemoteContent dangerouslySetInnerHTML={{ __html: content }} onInternalLinkClick={onInternalLinkClick} />
+      <RemoteContent
+        dangerouslySetInnerHTML={{ __html: sanitize(content) }}
```

Comment:
I can no longer see the previous discusison on this. Do we want to remove the sanitization in the api-client? I think we should avoid it here and do all sanitisation in the api-client. That way we easily support native and web. Also there are other fields which could require sanitisation.

I'm open for both, just sanitisating twice it a little bit overkill.

Reply:
Removed it here and created a separate issue to check our currently sanitizing and add it/improve it wherever necessary,The cms team also mentioned that our current sanitizing is not good enough.
Issue: https://issues.tuerantuer.org/browse/IGAPP-619